buhockeystatssite/formatresults.py

```diff
@@ -0,0 +1,148 @@
+    "St. Lawrence": ("#AF1E2D", "#4B2B23"),
+    "Colorado College": ("#000000", "#EFAB1E"),
+    "Union": ("#822433", "#FFFFFF"),
+    "Michigan State": ("#173F35", "#FFFFFF"),
+    "North Dakota": ("#009A44", "#FFFFFF"),
+    "Minnesota Duluth": ("#7A0019", "#FFCC33"),
+    "American International": ("#000000", "#FFB60F"),
+    "Wisconsin": ("#C5050C", "#FFFFFF"),
+    "Denver": ("#BA0C2F", "#A89968"),
+    "Alaska": ("#1E59AE", "#FCD006"),
+    "Providence": ("#000000", "#FFFFFF"),
+    "Bemidji State": ("#004D44", "#FFFFFF"),
+    "Merrimack": ("#003768", "#F1C400"),
+    "Notre Dame": ("#0C2340", "#C99700"),
+    "Connecticut": ("#000E2F", "#FFFFFF"),
+    "Ohio State": ("#BA0C2F", "#A1B1B7"),
+    "Vermont": ("#005710", "#FFC20E"),
+    "RIT": ("#F76902", "#FFFFFF"),
+    "Holy Cross": ("#602D89", "#FFFFFF"),
+    "Lake Superior State": ("#003F87", "#FFC61E"),
+    "UMass Lowell": ("#0067B1", "#C8102E"),
+    "Air Force": ("#003594", "#B2B4B2"),
+    "Bowling Green": ("#4F2C1D", "#FF7300"),
+    "Minnesota State": ("#480059", "#F7E400"),
+    "Western Michigan": ("#532E1F", "#F1C500"),
+    "Maine": ("#003263", "#B0D7FF"),
+    "Ferris State": ("#BA0C2F", "#FFD043"),
+    "Quinnipiac": ("#0A2240", "#FFB819"),
+    "Northern Michigan": ("#095339", "#FFC425"),
+    "Bentley": ("#1B5FAA", "#FFFFFF"),
+    "Miami": ("#B61E2E", "#FFFFFF"),
+    "Stonehill": ("#2F2975", "#FFFFFF"),
+    "Canisius": ("#0E2756", "#FFBA00"),
+    "Alaska Anchorage": ("#00583D", "#FFC425"),
+    "Mercyhurst": ("#0F4F44", "#FFFFFF"),
+    "Sacred Heart": ("#CD1041", "#FFFFFF"),
+    "Niagara": ("#582C83", "#FFFFFF"),
+    "Nebraska Omaha": ("#000000", "#D71920"),
+    "Penn State": ("#001E44", "#FFFFFF"),
+    "Robert Morris": ("#001E41", "#AA182C"),
+    "Arizona State": ("#7D2248", "#FFC72C"),
+    "Long Island": ("#69B3E7", "#FFC72C"),
+    "Lindenwood": ("#000000", "#B5A36A"),
+    "Augustana": ("#002D62", "#FFDD00"),
+    "Tufts": ("#3E8EDE", "#512C1D"),
+    "MIT": ("#231F20", "#B20D35"),
+    "Toronto": ("#002A5C", "#FFFFFF"),
+    "Colby": ("#254893", "#FFFFFF"),
+    "Dalhousie": ("#242424", "#FFD400"),
+    "Pennsylvania": ("#990000", "#011F5B"),
+    "St. Louis": ("#003DA5", "#C8C9C7"),
+    "Norwich": ("#9C0A2E", "#B99457"),
+    "Queen's": ("#B90E31", "#FABD0F"),
+    "Middlebury": ("#37538C", "#FFFFFF"),
+    "Williams": ("#362061", "#FECC06"),
+    "Hamilton College": ("#002F86", "#D6BA8B"),
+    "St. Francis Xavier": ("#0C2340", "#FFFFFF"),
+    "Syracuse": ("#F76900", "#FFFFFF"),
+    "Wayne State": ("#0C5449", "#FFCC33")
+}
+
+
+def format_results(df_results):
+    table_data = df_results[['date', 'opponent', 'result', 'scoreline', 'ot', 'arena', 'tourney', 'note']].copy()
+    table_data.fillna('', inplace=True)
+
+    table_data['date'] = table_data['date'].dt.strftime('%m/%d/%Y')
+
+    table_data.loc[table_data['ot'].notnull(), 'scoreline'] = table_data['scoreline'] + " (" + table_data.loc[table_data['ot'].notnull(), 'ot'] + ")"
```

Comment:
this will no longer work because table_data['ot'] no longer has null entries because of the fillna on line 114

Reply:
Should have it fixed now
<img width="603" alt="image" src="https://github.com/redsoxfan2194/buhockeystatsbot/assets/6944977/054b66bc-e9c6-4a8e-9b38-0e3ae617ff8b">